src/sdk/modules/smartSessionsValidator/decorators/findTrustedAttesters.ts

```diff
@@ -0,0 +1,64 @@
+import { getAccount } from "@rhinestone/module-sdk"
+import type { Address, Chain, Client, PublicClient, Transport } from "viem"
+import { parseAccount } from "viem/utils"
+import { AccountNotFoundError } from "../../../account/utils/AccountNotFound"
+import type { ModularSmartAccount } from "../../utils/Types"
+import { getTrustedAttesters } from "../Helpers"
+
+/**
+ * Parameters for retrieving the attesters of a smart account.
+ *
+ * @template TModularSmartAccount - The type of the smart account, which can be a ModularSmartAccount or undefined.
+ */
+export type FindTrustedAttestersParameters<
+  TModularSmartAccount extends ModularSmartAccount | undefined
+> = {
+  /** The smart account to get the attesters for. If not provided, the client's account will be used. */
+  account?: TModularSmartAccount
+}
+
+// Review: findTrustedAttesters also seems to be available in Rhinestone module sdk!
```

Comment:
these methods seem in rhinestone module-sdk as well. I haven't checked the implementation but we could use those instead.

Reply:
Let's use rhinestone's impl, the reason of using that package is to make our life easier because we won't have to write this kind of code ourselves.

Reply:
let me see if it fits well. some methods we have still implemented our own version cause their's were complicated

Reply:
ok I can do that.

Reply:
done